CondFormats/PCLConfig/plugins/AlignPCLThresholdsWriter.cc

```diff
@@ -65,10 +65,10 @@ AlignPCLThresholdsWriter::AlignPCLThresholdsWriter(const edm::ParameterSet& iCon
       m_minNrecords(iConfig.getParameter<unsigned int>("minNRecords")),
       m_parameters(iConfig.getParameter<std::vector<edm::ParameterSet> >("thresholds")) {
   //now do what ever initialization is needed
-  myThresholds = new AlignPCLThresholds();
```

Comment:
You can also use the empty initializer list constructor here

Reply:
ok will do!

Reply:
done in dfbacef